src/spec-node/devContainersSpecCLI.ts

```diff
@@ -803,6 +805,73 @@ async function doExec({
 	}
 }
 
+function mergeOptions(y: Argv) {
+	return y.options({
+		'parent-devcontainer': { type: 'string', description: 'Parent devcontainer path.' },
+		'child-devcontainer': { type: 'string', description: 'Child devcontainer path.' },
+	});
```

Comment:
I think this is taking a slightly different approach than what we've discussed with `"extends"` by having a parent and child declared as CLI arguments. The idea is that the devcontainer file itself would model this relationship with the `"extends"` keyword, e.g. 

```jsonc
// base.json
{
  "name": "Example",
  "ports": [80]
}

// devcontainer.json
{
  "extends": "./base.json",
  "ports": [22]
}
```

The CLI would resolve the "extends" keyword

```sh
$ devcontainer-cli read-configuration devcontainer.json
{
  "name": "Example",
  "ports": [22, 80]
}
```